puppet/modules/jenkins_job_builder/files/theforeman.org/yaml/builders/gemset_create.yaml

```diff
@@ -0,0 +1,4 @@
+- builder:
+    name: gemset_create
+    builders:
+      - shell: !include-raw scripts/gemset_create.sh
```

Comment:
First test of this, we'll need bundler present which means I can:

1) Create a builder that installs bundler into the RVM environment for job configurations to use
2) Update the slave manifest to ensure bundler is automatically installed into the base RVM that the gemsets use